src/applications/PseudoProgressionEstimator.h

```diff
@@ -393,6 +394,191 @@ class PseudoProgressionEstimator
+"T1_PCA_4","T1_PCA_5","T1_PCA_6","T1_PCA_7","T1_PCA_8","T1_PCA_9","T1_PCA_10","T1CE_PCA_1","T1CE_PCA_2","T1CE_PCA_3",
+"T1CE_PCA_4","T1CE_PCA_5","T1CE_PCA_6","T1CE_PCA_7","T1CE_PCA_8","T1CE_PCA_9","T1CE_PCA_10","T1T1CE_PCA_1","T1T1CE_PCA_2",
+"T1T1CE_PCA_3","T1T1CE_PCA_4","T1T1CE_PCA_5","T1T1CE_PCA_6","T1T1CE_PCA_7","T1T1CE_PCA_8","T1T1CE_PCA_9","T1T1CE_PCA_10",
+"T2_PCA_1","T2_PCA_2","T2_PCA_3","T2_PCA_4","T2_PCA_5","T2_PCA_6","T2_PCA_7","T2_PCA_8","T2_PCA_9","T2_PCA_10","FL_PCA_1",
+"FL_PCA_2","FL_PCA_3","FL_PCA_4","FL_PCA_5","FL_PCA_6","FL_PCA_7","FL_PCA_8","FL_PCA_9","FL_PCA_10","T2FL_PCA_1","T2FL_PCA_2",
+"T2FL_PCA_3","T2FL_PCA_4","T2FL_PCA_5","T2FL_PCA_6","T2FL_PCA_7","T2FL_PCA_8","T2FL_PCA_9","T2FL_PCA_10","AX_PCA_1",
+"AX_PCA_2","AX_PCA_3","AX_PCA_4","AX_PCA_5","AX_PCA_6","AX_PCA_7","AX_PCA_8","AX_PCA_9","AX_PCA_10","FA_PCA_1","FA_PCA_2",
+"FA_PCA_3","FA_PCA_4","FA_PCA_5","FA_PCA_6","FA_PCA_7","FA_PCA_8","FA_PCA_9","FA_PCA_10","RAD_PCA_1","RAD_PCA_2","RAD_PCA_3",
+"RAD_PCA_4","RAD_PCA_5","RAD_PCA_6","RAD_PCA_7","RAD_PCA_8","RAD_PCA_9","RAD_PCA_10","TR_PCA_1","TR_PCA_2","TR_PCA_3",
+"TR_PCA_4","TR_PCA_5","TR_PCA_6","TR_PCA_7","TR_PCA_8","TR_PCA_9","TR_PCA_10","PH_PCA_1","PH_PCA_2","PH_PCA_3","PH_PCA_4",
+"PH_PCA_5","PH_PCA_6","PH_PCA_7","PH_PCA_8","PH_PCA_9","PH_PCA_10","PSR_PCA_1","PSR_PCA_2","PSR_PCA_3","PSR_PCA_4",
+"PSR_PCA_5","PSR_PCA_6","PSR_PCA_7","PSR_PCA_8","PSR_PCA_9","PSR_PCA_10","RCBV_PCA_1","RCBV_PCA_2","RCBV_PCA_3",
+"RCBV_PCA_4","RCBV_PCA_5","RCBV_PCA_6","RCBV_PCA_7","RCBV_PCA_8","RCBV_PCA_9","RCBV_PCA_10","PCA1_PCA_1","PCA1_PCA_2",
+"PCA1_PCA_3","PCA1_PCA_4","PCA1_PCA_5","PCA1_PCA_6","PCA1_PCA_7","PCA1_PCA_8","PCA1_PCA_9","PCA1_PCA_10","PCA2_PCA_1",
+"PCA2_PCA_2","PCA2_PCA_3","PCA2_PCA_4","PCA2_PCA_5","PCA2_PCA_6","PCA2_PCA_7","PCA2_PCA_8","PCA2_PCA_9","PCA2_PCA_10",
+"PCA3_PCA_1","PCA3_PCA_2","PCA3_PCA_3","PCA3_PCA_4","PCA3_PCA_5","PCA3_PCA_6","PCA3_PCA_7","PCA3_PCA_8","PCA3_PCA_9",
+"PCA3_PCA_10","PCA4_PCA_1","PCA4_PCA_2","PCA4_PCA_3","PCA4_PCA_4","PCA4_PCA_5","PCA4_PCA_6","PCA4_PCA_7","PCA4_PCA_8",
+"PCA4_PCA_9","PCA4_PCA_10","PCA5_PCA_1","PCA5_PCA_2","PCA5_PCA_3","PCA5_PCA_4","PCA5_PCA_5","PCA5_PCA_6","PCA5_PCA_7",
+"PCA5_PCA_8","PCA5_PCA_9","PCA5_PCA_10","PCA6_PCA_1","PCA6_PCA_2","PCA6_PCA_3","PCA6_PCA_4","PCA6_PCA_5","PCA6_PCA_6",
+"PCA6_PCA_7","PCA6_PCA_8","PCA6_PCA_9","PCA6_PCA_10","PCA7_PCA_1","PCA7_PCA_2","PCA7_PCA_3","PCA7_PCA_4","PCA7_PCA_5",
+"PCA7_PCA_6","PCA7_PCA_7","PCA7_PCA_8","PCA7_PCA_9","PCA7_PCA_10","PCA8_PCA_1","PCA8_PCA_2","PCA8_PCA_3","PCA8_PCA_4",
+"PCA8_PCA_5","PCA8_PCA_6","PCA8_PCA_7","PCA8_PCA_8","PCA8_PCA_9","PCA8_PCA_10","PCA9_PCA_1","PCA9_PCA_2","PCA9_PCA_3",
+"PCA9_PCA_4","PCA9_PCA_5","PCA9_PCA_6","PCA9_PCA_7","PCA9_PCA_8","PCA9_PCA_9","PCA9_PCA_10","PCA10_PCA_1","PCA10_PCA_2",
+"PCA10_PCA_3","PCA10_PCA_4","PCA10_PCA_5","PCA10_PCA_6","PCA10_PCA_7","PCA10_PCA_8","PCA10_PCA_9","PCA10_PCA_10" };
+
```

Comment:
A block of string literals this big hurts me to look at. Since these seem to generally be repeating across modalities/principal component numbers, maybe we should generate this list programmatically?

Reply:
So I did discuss this with @saimarathore . I do agree that doing this programmatically would be much better. Even better to have parts of this enumerated and manage this in a centralized manner. However, the amount of effort required to do this programmatically would be much higher and would take up substantial time.

Reply:
Fair enough. If this PR should get into the next release, we should add this as a separate issue for later. I'll approve this.